### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ARTI1208**

### CODE REVIEW

**Comment by ARTI1208** [comment]
I'm not sure about licenses it gradlew and gradlew.bat. @SerVB did you modify them the last time you updated wrapper?

**Review by SerVB — CHANGES_REQUESTED** [review]

**Comment by SerVB** [comment]
> I'm not sure about licenses it gradlew and gradlew.bat. @SerVB did you modify them the last time you updated wrapper?

I remember some day IDEA inserted licenses to `gradlew`s and I removed them manually. But it was long ago and I don't remember the details.

When I update Gradle, I just change the version in `gradle/wrapper/gradle-wrapper.properties`, I don't update the scripts. If you think scripts should be updated too, feel free to do it. Regarding licenses, please just compare these files with original files...

**Comment by ARTI1208** [comment]
> IDEA inserted licenses

It looks it's not the case now, licenses appeared when I ran gradlew command to update wrapper


> I just change the version

For some reason it didn't work for me this time, but usually I do the same way

> Regarding licenses, please just compare these files with original files

What do you mean saying "original" files?

**Comment by ARTI1208** [comment]
Are integration tests broken?

File: projector-server-core/build.gradle.kts [hunk 1]
```diff
@@ -140,12 +140,22 @@ dependencies {
   implementation("dnsjava:dnsjava:$dnsjavaVersion")
   implementation("org.jsoup:jsoup:$jsoupVersion")
 
+  fun ModuleDependency.setupMarkdownDependency() {
+    exclude(group = "ai.grazie.utils")
+      .exclude(group = "ai.grazie.nlp")
+      .exclude(group = "ai.grazie.spell")
+      .exclude(group = "com.jetbrains.infra")
```
[the remark this answers is not in the document]
Comment (SerVB) [reply projector-server-core/build.gradle.kts]: I don't know either. The changes you've just made LGTM, let's resolve this.

**Comment by SerVB** [comment]
Probably because you've changed them. Do they work for you locally?

Btw, `ClickOptions.usingDefaultMethod()` seems suspicious: before it was `(5, 5)`. Maybe something like `ClickOptions.offset(5, 5)` will match the previous logic.

**Review by SerVB — APPROVED** [review]
@Avvessalom , could you please help here?
1. With int tests failing.
2. Qodana reports places unrelated to this PR, maybe we can submit it as a piece of feedback to the Team.

**Comment by ARTI1208** [comment]
> Btw, ClickOptions.usingDefaultMethod() seems suspicious
These integration tests are passing with this param. `(5, 5)` was only needed to avoid using `click()` without arguments (as suggests comment), so `(5, 5)` and `ClickOptions.usingDefaultMethod()` essentialy play the same role here

**Review by Avvessalom — APPROVED** [review]

**Pull request merged**